dfastmi/io/RiverParameterData.py

```diff
@@ -49,25 +51,26 @@ class RiverParameterData:
     """ 
         The dictionary containing river data.
     """
-    _is_initialized : bool
+    #_is_initialized : bool
 
     def __init__(self, config: configparser.ConfigParser):
         self._config = config
-        self._is_initialized = False
+    #    self._is_initialized = False
     
-    def initialize(self, branches: List[str], nreaches: List[int]) : 
-        """
-            This routine initialize the object with required objects
-        """
-        self._branches = branches
-        self._nreaches = nreaches
-        self._is_initialized = True
+    #def initialize(self, branches: List[str], nreaches: List[int]) : 
```

Comment:
whats the plan with this commented part?

Reply:
will be removed

Reply:
Discussed, we will leave these comments for a later refactoring.
This will be tackled in https://issuetracker.deltares.nl/browse/DFAST-112